lib/chat_app_web/live/single_room_live.html.heex

```diff
@@ -0,0 +1,80 @@
+<.header>
+  <div>
+    <% room_name = @room.room_name %>
+    Room "<%= room_name %>"
+    <div class="float-right">
+      <button class="rounded-2xl hover:animate-pulse hover:motion-reduce bg-purple-800 p-1 px-5 text-white text-base peer">Users</button>
+      <aside class="border border border-emerald-500 z-10 grid fixed top-0 bg-emerald-100 w-[233px] h-full -left-[233px] peer-focus:left-0 ease-out delay-150 duration-300 rounded-r-[25px]">
+        <h1 class="text-purple-800 bold text-[15px] mt-8 text-center uppercase"> Users in room "<%= room_name %>"</h1>
+        <ul>
+          <% number_of_users = length(@users) %>
+          <%= if number_of_users > 0 do %>
+            <%= for user_index <- 0..number_of_users-1 do %>
+              <li class="p-2 mx-5 text-[13px] hover:bg-emerald-200 rounded-2xl"><%=@users |> Enum.at(user_index)%></li>
+            <% end %>
+          <% end %>
+        </ul>
+      </aside>
+    </div>
+  </div>
+</.header>
+
+
+<div id="chatbox" class="h-80 rounded-2xl bg-green-100 overflow-x-auto overflow-y-auto mt-3">
+  <ul>
+    <% number_of_messages = length(@messages) %>
+    <%= if number_of_messages > 0 do %>
+      <%= for message_index <- number_of_messages-1..0 do %>
+        <%
+          {message_id, actual_message} = Enum.at(@messages, message_index)
+          %ChatApp.Message{text: text, sender: sender, room_id: _, timestamp: timestamp, is_deleted: is_deleted, is_edited: is_edited} = actual_message
```

Comment:
Why do we need is_edited as well?

Reply:
Well, I thought it would be a good idea to show on the UI if the message was edited or not. If is_edited is true, then the UI will display an "(edited)" tag near the edited message.